schema/askbot.csv

```diff
@@ -14,6 +15,7 @@ author_uuid,keyword
 comment_count,long
 first_answer,long
 grimoire_creation_date,date
+has_accepted_answer,long

```

Comment:
I would change the name to `question_has_accepted_answer` just to follow the same pattern we re using below to identify question related fields.